src/store/index.js

```diff
@@ -112,13 +112,23 @@ export default new Vuex.Store({
             state.gameState.playerInfo = !state.gameState.playerInfo
         },
         stepTwoToQuestPhase: ({state}) => {
-            state.gameState.playerInfo = !state.gameState.playerInfo
-            state.gameState.questInfoDisplay = !state.gameState.questInfoDisplay
-            state.gameState.proposeMissionParty = !state.gameState.proposeMissionParty
+            //This means that we're already past the ProposeMissionParty state so we jump straight to ProposedPartyVote
+            if (state.ProposedPartyVoteMenu.proposedParty.length >= 1) {
```

Comment:
is this conditional doing anything?

Reply:
seems to me like you can drop the if else and just keep

>                 state.gameState.playerInfo = false
>                 state.gameState.questInfoDisplay = true
>                 state.gameState.proposedPartyVote = true

Reply:
@blarmon This conditional is checking if we have received a proposed party. Since `stepTwoToQuestPhase` is called when the player clicks ready on the `PlayerInfo` screen, we need to know which state we are going to. If we have received a proposed party we have to enable `proposedPartyVote`, and if we haven't then we enable the `proposeMissionParty`. Let me know if that makes sense! Similar to your other comment, I could move this to the `PlayerInfo` component if you think that makes more sense.

Reply:
@blarmon Let me know and I can do this in a separate PR. Since everything else has been addressed I'm gonna get this moving. I'd like to deploy the backend changes and that'll break things if this isn't also in production, sadly 😞

Reply:
sorry, I misread it.  you good.